docs/wdc_debugging.md

```diff
@@ -73,6 +72,8 @@ connector. Here are some ways to work around caching issues:
 
 ### General browser debugging tips {#browser-tips}
 
+<!-- Not sure how this was supposed to work in with for WebKit-based debugging, these instructions don't say how you connect to the WDC -->
```

Comment:
is this comment staying?

Reply:
Opps! No.  I thought you'd catch that. but I forgot to remove it.